annotation/tests/test_distribution.py

```diff
@@ -1516,3 +1524,144 @@ def test_prepare_response():
             )
             == expected_output
         )
+
+
+@pytest.fixture
+def mock_session_and_tasks():
+    task_list = [
+        ManualAnnotationTask(
+            id=0,
+            status=TaskStatusEnumSchema.finished,
+            is_validation=True,
+            file_id=0,
+            pages=[1, 2, 3, 4],
+        ),
+        ManualAnnotationTask(
+            id=1,
+            status=TaskStatusEnumSchema.finished,
+            is_validation=False,
+            file_id=1,
+            pages=list(range(20)),
+        ),
+    ]
+
+    mock_db = Mock(spec=Session)
+    mock_query = mock_db.query.return_value
+    mock_filter = mock_query.filter.return_value
+    mock_all = mock_filter.all
+    mock_all.return_value = task_list
```

Comment:
I think you can merge this into one line no need to put it into multiple lines

Reply:
Yes, it can be shortened. It'll be fixed